docs/ske/001-quick-start.md

```diff
@@ -8,11 +8,25 @@ sidebar_label: Quick Start (Enterprise)
 import useBaseUrl from '@docusaurus/useBaseUrl';
 ```
 
-
 :::info
 
-In order to run through this page you need Syntasso Kratix Enterprise license **token**.
-Don't have one? [Request your free trial token here](https://www.syntasso.io/pricing)
+This guide walks you through a **quick-start, development-only installation** of
```

Comment:
Same here, this info section should come after the title ("_Quick Start (Enterprise Edition)_").